base/audit_events.go

```diff
@@ -636,13 +636,17 @@ var AuditEvents = events{
 		Name:        "Replication connect",
 		Description: "A replication client connected",
 		MandatoryFields: AuditFields{
-			"db":  "database name",
-			"cid": "context id",
+			"cid":         "context id", // this is blip context id
```

Comment:
Is this different than the correlation ID (present in fieldGroupRequest)

Reply:
This is not different, this is actually a duplicate. As discussed offline, we are going to rely on the difference in the format:

blip `"[" + contextID + "]"` and HTTP `#%03d`. I will remove this.